cmake/export.cmake

```
@@ -12,50 +12,50 @@
 
 include(CMakePackageConfigHelpers)
 
-write_basic_package_version_file(gt_c_bindings-config-version.cmake
+write_basic_package_version_file(cpp_bindgen-config-version.cmake
     VERSION ${PROJECT_VERSION}
     COMPATIBILITY SameMajorVersion
     )
 
-set(gt_c_bindings_MODULE_PATH lib/cmake)
-set(gt_c_bindings_SOURCES_PATH src)
-set(gt_c_bindings_INCLUDE_PATH include)
+set(cpp_bindgen_MODULE_PATH lib/cmake)
+set(cpp_bindgen_SOURCES_PATH src)
+set(cpp_bindgen_INCLUDE_PATH include)
 
-configure_package_config_file(cmake/gt_c_bindings-config.cmake.in
-${CMAKE_CURRENT_BINARY_DIR}/gt_c_bindings-config.cmake
-PATH_VARS gt_c_bindings_MODULE_PATH gt_c_bindings_SOURCES_PATH gt_c_bindings_INCLUDE_PATH
+configure_package_config_file(cmake/cpp_bindgen-config.cmake.in
+${CMAKE_CURRENT_BINARY_DIR}/cpp_bindgen-config.cmake
+PATH_VARS cpp_bindgen_MODULE_PATH cpp_bindgen_SOURCES_PATH cpp_bindgen_INCLUDE_PATH
 INSTALL_DESTINATION lib/cmake
 )
 
 install(
     FILES
-    ${CMAKE_CURRENT_BINARY_DIR}/gt_c_bindings-config.cmake
-    ${CMAKE_CURRENT_BINARY_DIR}/gt_c_bindings-config-version.cmake
+    ${CMAKE_CURRENT_BINARY_DIR}/cpp_bindgen-config.cmake
+    ${CMAKE_CURRENT_BINARY_DIR}/cpp_bindgen-config-version.cmake
     DESTINATION lib/cmake
     )
     
-set(__C_BINDINGS_CMAKE_DIR "\${gt_c_bindings_MODULE_PATH}")  #TODO refactor the variable names gt_c_bindings_MODULE_PATH, etc.
-set(__C_BINDINGS_SOURCE_DIR "\${gt_c_bindings_SOURCES_PATH}")
-set(__C_BINDINGS_INCLUDE_DIR "\${gt_c_bindings_INCLUDE_PATH}")
-configure_file(cmake/gt_bindings.cmake.in
-    ${PROJECT_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/build-install/lib/cmake/gt_bindings.cmake
+set(__C_BINDINGS_CMAKE_DIR "\${cpp_bindgen_MODULE_PATH}")  #TODO refactor the variable names cpp_bindgen_MODULE_PATH, etc.
```

```suggestion
set(__C_BINDINGS_CMAKE_DIR "\${cpp_bindgen_MODULE_PATH}")
```
I think that's what this PR is intended for, right?

---

No this TODO is for syncing the names. Sometimes we call it MODULE_PATH, sometimes CMAKE_DIR, etc.